x-pack/plugins/session_view/common/types/process_tree/index.ts

```diff
@@ -15,20 +15,36 @@ export enum EventAction {
   exec = 'exec',
   exit = 'exit',
   output = 'output',
+  connect = 'connect',
 }
 
 export interface EventActionPartition {
   fork: ProcessEvent[];
   exec: ProcessEvent[];
   exit: ProcessEvent[];
   output: ProcessEvent[];
+  connect: ProcessEvent[];
```

Comment:
Type connect? Where is the sourth of truth for this?

Reply:
The connect action was showing up in my response but I didn't dig into it until now. It was a custom event action defined in event converter https://github.com/Elastic-AWP-Platform/EventConverter/blob/main/src/main.ts#L18, but it won't to applicable to endpoint or us. I will remove this. thanks for pointing it out

Reply:
Ah yea I see. We should probably not do anything with the CONNECT and DISCONNECT messages in event converter. Only FORK, EXEC, and EXIT events should be inserted at this point. The connect event from the agent is legacy and won't have any of the exec info needed to populate a session leader.